python/cudf/cudf/_cuda/gpu.pyx

```diff
@@ -341,7 +341,7 @@ def deviceGetName(int device):
     """
 
     cdef char* device_name = <char*> malloc(256 * sizeof(char))
-    status = cuDeviceGetName(device_name, 256, device)
+    cdef int status = cuDeviceGetName(device_name, 256, device)
```

Comment:
Noted we could clean this up a bit as well. Submitted PR ( https://github.com/rapidsai/cudf/pull/4927 ) to do that.